JFS_CAN_report_instructions.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "JFS CAN Data Report Instructions"
+format: pdf
+editor: visual
+---
+
+## Open the Starter App
+
+1.  In Docker Desktop, search for "Degauss/shiny_starter"
+
+    A. "Pull" the container
+
+2.  Once the container is in your Desktop, find it and hit "Run"
+
+    A. In the popup box, expand the options and in the space marked "Local Host", type in "3838" and hit Run
+
+3.  Navigate to the Containers page and in your line for the shiny_starter, click to button to open in browser
+
+4.  Now that you're in the app, you can upload your .csv file
+
+5.  After the upload, the app will show you a sample of your original file and the prepared file for use in the next program.
+
+6.  Click the button to download the prepped file
+
+7.  Now in PowerShell, navigate to your Downloads folder (it may be as simple as typing "cd Downloads")
+
+8.  Copy & paste the first Docker command into PowerShell and hit enter
```

Comment:
Just thinking that Kris could also copy and paste these commands into a PowerShell script and run that everytime she needs to execute the workflow. (Not sure if she uses scripts like that or not...?)

Reply:
I think she just copies/pastes each individually in there, so I don't know if we want to mess with that too much :)